index.html

```diff
@@ -246,6 +247,24 @@ <h4>UX Designer</h4>
           </div>
         </div>
       </section>
+      <section class="leave-a-message">
```

Comment:
class name `"message"` for short would be better, the classes are for styling an don't need to be very descriptive
e.g. `leave-message` would be good for `id` if use as an anchor link